datajunction-server/tests/api/dimension_links_test.py

```diff
@@ -326,13 +326,19 @@ async def test_link_complex_dimension_without_role(
     assert str(parse(query)) == str(parse(expected))
 
     response = await dimensions_link_client.get("/nodes/default.events/dimensions")
-    assert [(attr["name"], attr["path"]) for attr in response.json()] == [
-        ("default.users.account_type", ["default.events"]),
-        ("default.users.registration_country", ["default.events"]),
-        ("default.users.residence_country", ["default.events"]),
-        ("default.users.snapshot_date", ["default.events"]),
-        ("default.users.user_id", ["default.events"]),
-    ]
+    assert sorted(
+        [(attr["name"], attr["path"]) for attr in response.json()],
+        key=lambda x: x[0],
+    ) == sorted(
+        [
+            ("default.users.account_type", ["default.events", "default.users"]),
```

Comment:
I am trying to figure out why the `default.users` item was added here. Any ideas?